tests/pw/.env.bak

```
@@ -0,0 +1,36 @@
+ADMIN=shohan
+ADMIN_PASSWORD=01dokan01
+VENDOR=vendor1
+VENDOR2=vendor2
+CUSTOMER=customer1
+CUSTOMER2=customer2
+USER_PASSWORD=01dokan01
+
+# Dokan Configuration
+DOKAN_PRO=true
+LICENSE_KEY=74c8654e-84a5-49b2-a3f5-81afba8e13fa
+GMAP=AIzaSyD9N67E6zpGuZqT-o_EI8da5qLbWonLOWw
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GMAP key should go before the LICENSE_KEY key

(UnorderedKey)